Tests/SonarQube.Common.UnitTests/ProcessRunnerTests.cs

```diff
@@ -264,7 +264,9 @@ public void ProcRunner_DoNotLogSensitiveData()
             {
                 "public1",
                 "public2",
-                "/d:sonar.projectKey=my.key"
+                "/d:sonar.projectKey=my.key",
+                "/d:sonar.secured.notreallysecured=key",
+                "/d:sonarsecured=key",
```

Comment:
Additional public test cases (should be able to pass anything in the value part):
/d:sonar.public=foo.secured
/d:sonar.public2=sonar.password